modules/monitoring-common-terms.adoc

```diff
@@ -0,0 +1,93 @@
+
+custom resource (CR)::
+A CR is extension of the Kubernetes API. You can create custom resources.
+
+etcd::
+etcd is the key-value store for OpenShift Container Platform, which stores the state of all resource objects.
+
+Fluentd::
+Fluentd gathers logs from nodes and feeds them to Elasticsearch.
+
+Kubelets::
+Runs on nodes and reads the container manifests. Ensures that the defined containers have started and are running.
+
+Kubernetes API server::
+Kubernetes API server validates and configures data for the API objects.
+
+Kubernetes controller manager::
+Kubernetes controller manager governs the state of the cluster.
+
+Kubernetes scheduler::
+Kubernetes scheduler allocates pods to nodes.
+
+labels::
+You can use labels, which are key-value pairs, to organise and select subsets of objects, such as a pod.
+
+Metering::
+Metering is a general purpose data analysis tool that enables you to write reports to process data from different data sources.
+
+node::
+A worker machine in the {product-title} cluster. A node is either a virtual machine (VM) or a physical machine.
+
+Operator::
+The preferred method of packaging, deploying, and managing a Kubernetes application in an {product-title} cluster. An Operator takes human operational knowledge and encodes it into software that is packaged and shared with customers.
+
+Operator Lifecycle Manager (OLM)::
+OLM helps you to install, update, and manage the lifecycle of Kubernetes native applications. OLM is an open source toolkit designed to manage Operators in an effective, automated, and scalable way.
+
+Persistent storage::
+Stores the data even after the device is shut down. Kubernetes uses persistent volumes to store the application data
+
+Persistent volume claim (PVC)::
+You can use a PVC to mount a PersistentVolume into a Pod. You can access the storage without knowing the details of the cloud environment.
+
+pod::
+The pod is the smallest logical unit in Kubernetes. A pod contains one or more containers to run in a worker node.
+
+Prometheus::
+Prometheus is the monitoring system on which the OpenShift Container Platform monitoring stack is based. Prometheus is a time-series database and a rule evaluation engine for metrics. Prometheus sends alerts to Alertmanager for processing.
+
+Prometheus adapter::
+The Prometheus Adapter translates Kubernetes node and pod queries for use in Prometheus. The resource metrics that are translated include CPU and memory utilization metrics. The Prometheus Adapter exposes the cluster resource metrics API for horizontal pod autoscaling.
+
+Prometheus Operator::
+The Prometheus Operator (PO) in the `openshift-monitoring` project creates, configures, and manages platform Prometheus instances and Alertmanager instances. It also automatically generates monitoring target configurations based on Kubernetes label queries.
```

Comment:
If Prometheus and Altermanager are both platforms, then you could omit one of the "instances".

Reply:
acked